resources/dicts/relationship_events/normal_interactions/trust/increase.json

```diff
@@ -1,349 +1,400 @@
+			"m_c promises to always look out for r_c.",
+			"m_c trusts r_c with a very important task.",
+			"m_c often goes to r_c for guidance.",
+			"r_c admits a mistake to m_c and asks for forgiveness.",
+			"r_c believes in m_c's version of events when something happens and the Clan doubts {PRONOUN/m_c/object}.",
+			"m_c backs up one of r_c's claims during one of r_c's arguments.",
+			"m_c and r_c agree on something the rest of the Clan finds contentious.",
+			"m_c tells r_c to count on {PRONOUN/m_c/object} in a difficult situation."
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_highpat",
+		"interactions": [
+			"m_c and r_c watch one another's back on a patrol.",
+			"m_c protects r_c from something on a patrol.",
+			"m_c and r_c chase off a predator on a patrol together.",
+			"m_c and r_c make an elaborate plan to evict a predator from the Clan's territory.",
+			"m_c and r_c volunteer to go on an extra border patrol together.",
+			"m_c notices that r_c has finished all {PRONOUN/r_c/poss} tasks without issues lately.",
+			"During a patrol, m_c and r_c have a long conversation about their beliefs and agree on many things.",
+			"During the last gathering, m_c and r_c defended their Clan against another Clan's heckling."
+		],
+		"main_status_constraint": [
+			"apprentice",
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"medicine cat apprentice",
+			"leader"
+		],
+		"random_status_constraint": [
+			"apprentice",
+			"warrior",
+			"mediator",
+			"medicine cat",
+			"deputy",
+			"leader",
+			"medicine cat apprentice"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_high3",
+		"intensity": "high",
+		"interactions": [
+			"m_c tells r_c a deep, dark secret.",
+			"m_c confesses something to r_c.",
+			"m_c depends on r_c to back {PRONOUN/m_c/object} up during a debate.",
+			"m_c and r_c swap deep, dark secrets with each other."
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"reaction_random_cat": {
+			"trust": "increase"
+		}
+	},
+	{
+		"id": "trust_inc_high4",
+		"intensity": "high",
+		"interactions": [
+			"m_c goes to r_c for advice about some recent troubles.",
+			"m_c asks r_c to help {PRONOUN/m_c/object} find the right words to say to a Clanmate.",
+			"m_c asks r_c for help resolving an argument with a friend."
+		],
+		"relationship_constraint": [
+			"trust_30"
+		],
+		"random_status_constraint": [
+			"elder",
+			"mediator",
+			"mediator apprentice"
+		]
+	},
+	{
+		"id": "trust_star_inc_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c tells r_c about {PRONOUN/m_c/poss} last encounter with StarClan.",
+			"m_c debates about the meaning of a prophetic dream with r_c.",
+			"r_c puts a lot of faith into m_c's skill as {PRONOUN/m_c/poss} rank."

```

Comment:
this feels a bit clunky. I'd personally recommend removing the part about their rank, or having separate versions for medicine cats and leaders